docs/resources/Guild.md

```diff
@@ -94,6 +95,14 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 | HIGH      | 3       | must be a member of the server for longer than 10 minutes |
 | VERY_HIGH | 4       | must have a verified phone number                         |
 
+###### Guild NSFW Level
+| Level          | Integer |
```

Comment:
```suggestion
| Level          | Value |
```

Reply:
docs are pretty inconsistent about this, a bunch of tables do use"Integer"

Reply:
> docs are pretty inconsistent about this, a bunch of tables do use "Integer"

Then they should also be corrected. I could probably open another PR for that sometime during the weekend.